kill.c

```diff
@@ -41,6 +41,16 @@ static int isnumeric(char* str)
 	}
 }
 
+static void maybe_notify(char* notif_command, char* notif_args)
+{
+	if(notif_command)
+	{
+		char notif[200 + sizeof notif_command];
```

Comment:
sizeof(notif_command) will return the size of the pointer, not the size of the string. In other words, it will always return 8 on 64-bit architectures.

Suggest to simply use hard-coded value 400.